orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/image/FindImageFromTagsTask.java

```diff
@@ -46,9 +46,11 @@ public TaskResult execute(Stage stage) {
       .orElseThrow(() -> new IllegalStateException("ImageFinder not found for cloudProvider " + cloudProvider));
 
     StageData stageData = (StageData) stage.mapTo(StageData.class);
+    Collection<ImageFinder.ImageDetails> imageDetails = imageFinder.byTags(stage, stageData.packageName, stageData.tags);
     return new DefaultTaskResult(
       ExecutionStatus.SUCCEEDED,
-      Collections.singletonMap("amiDetails", imageFinder.byTags(stage, stageData.packageName, stageData.tags))
+      Collections.singletonMap("amiDetails", imageDetails),
+      Collections.singletonMap("deploymentDetails", imageDetails)
```

Comment:
Not a fan of putting more stuff in the global context but this will require some work in `deck` to move away from (_it's an already established pattern so this isn't necessarily making anything worse_).